vertical-pod-autoscaler/pkg/decode-histogram/main.go

```diff
@@ -50,6 +51,21 @@ func main() {
 	aggregateState := model.NewAggregateContainerState()
 	// Load data into the AggregateContainerState using LoadFromCheckpoint
 	err = aggregateState.LoadFromCheckpoint(&checkpoint.Status)
+
+	layout := "2006-01-02 15:04:05.999999999 -0700 MST"
```

Comment:
Let's make the `layout` a constant and reuse it everywhere for both serialization and deserialization.